rotkehlchen/chain/evm/transactions.py

```diff
@@ -267,13 +268,14 @@ def _query_and_save_internal_transactions_for_range(
                     )
                 if len(result) == 0:  # parent transaction is not in the DB. Get it
                     transaction = self.evm_inquirer.get_transaction_by_hash(internal_tx.parent_tx_hash)  # noqa: E501
-                    with self.database.conn.write_ctx() as write_cursor:
-                        dbevmtx.add_evm_transactions(
-                            write_cursor=write_cursor,
-                            evm_transactions=[transaction],
-                            relevant_address=address,
-                        )
-                    timestamp = transaction.timestamp
+                    if transaction is not None:
```

Comment:
What are you doing here? The parent transaction **should not ever** be `None` here. If it is you have to log an error and get out of here